arch/arm/src/nrf52/nrf52_gpiote.h

```diff
@@ -72,6 +57,10 @@
 int nrf52_gpiosetevent(uint32_t pinset, bool risingedge, bool fallingedge,
                        bool event, xcpt_t func, FAR void *arg);
 
+int nrf52_gpio_taskset(uint32_t pinset, int channel,
```

Comment:
Let's stick to the naming convention that is already in use.
```suggestion
int nrf52_gpiotaskset(uint32_t pinset, int channel,
```

Reply:
Done